core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -344,6 +344,10 @@ public RelNode getRoot() {
     return root;
   }
 
+  public ImmutableList<RelOptMaterialization> getMaterialization() {
```

Comment:
@michaelmior : Yes, this needs to be plural. Just looked at the comment now. Had missed this comment earlier due to some messup I did with email filter for github . Sorry about that.